<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_amend</class>
 <widget class="QDialog" name="Dialog_amend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>330</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>修改项</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_L2">
       <property name="toolTip">
        <string>机床归零后，A轴到C轴的Y方向距离。正对机床，A轴在C轴左侧值为正，右侧值为负，理想情况下此值为0</string>
       </property>
       <property name="text">
        <string>L2</string>
       </property>
       <property name="buddy">
        <cstring>dSB_L2</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_L4">
       <property name="toolTip">
        <string>机床归零后，主轴轴线到A轴的Y方向距离。正对机床，主轴轴线在A轴左侧值为正，右侧值为负，理想情况下此值为0</string>
       </property>
       <property name="text">
        <string>L4</string>
       </property>
       <property name="buddy">
        <cstring>dSB_L4</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="dSB_L4">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-9999.998999999999796</double>
       </property>
       <property name="maximum">
        <double>9999.998999999999796</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="dSB_L2">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-9999.998999999999796</double>
       </property>
       <property name="maximum">
        <double>9999.998999999999796</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="dSB_L3">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-9999.998999999999796</double>
       </property>
       <property name="maximum">
        <double>9999.998999999999796</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_L1">
       <property name="toolTip">
        <string>机床归零后，主轴旋转轴线到C轴的X方向距离</string>
       </property>
       <property name="text">
        <string>L1</string>
       </property>
       <property name="buddy">
        <cstring>dSB_L1</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="dSB_L1">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-9999.998999999999796</double>
       </property>
       <property name="maximum">
        <double>9999.998999999999796</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_L3">
       <property name="toolTip">
        <string>机床归零后，主轴端面到A轴的Z方向距离</string>
       </property>
       <property name="text">
        <string>L3</string>
       </property>
       <property name="buddy">
        <cstring>dSB_L3</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>名称</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_name</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>dSB_L1</tabstop>
  <tabstop>dSB_L2</tabstop>
  <tabstop>dSB_L3</tabstop>
  <tabstop>dSB_L4</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_amend</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_amend</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
